src/win/pipe.c

```diff
@@ -471,6 +472,8 @@ void uv_pipe_endgame(uv_loop_t* loop, uv_pipe_t* handle) {
 
 
 void uv_pipe_pending_instances(uv_pipe_t* handle, int count) {
+  if (handle->flags & UV_HANDLE_BOUND)
+    return;
```

Comment:
Ah, I see. I'd go for doing it if need be, I doubt this will be a problem for anyone at this point. FWIW I had to dig what thing actually meant...